backend/compilers/download.py

```diff
@@ -161,6 +161,82 @@ def download_zip(
             f.extract(member=file, path=dest_path)
 
 
+def download_ppc_darwin():
+    if host_os != LINUX:
+        print("MAC OS X cross compiler unsupported on " + host_os.name)
+        return
+    download_tar(
+        url="https://github.com/ChrisNonyminus/powerpc-darwin-cross/releases/download/initial/gcc-5370.tar.gz",
+        dl_name="gcc-5370.tar.gz",
+        dest_name="gcc-5370",
+    )
+    download_tar(
+        url="https://github.com/ChrisNonyminus/powerpc-darwin-cross/releases/download/initial/gcc-5370.tar.gz",
+        dl_name="gcc-5370-cpp.tar.gz",
+        dest_name="gcc-5370-cpp",
+    )
+    download_tar(
+        url="https://github.com/ChrisNonyminus/powerpc-darwin-cross/releases/download/initial/gcc-5026.tar.gz",
+        dl_name="gcc-5026.tar.gz",
+        dest_name="gcc-5026",
+    )
+    download_tar(
+        url="https://github.com/ChrisNonyminus/powerpc-darwin-cross/releases/download/initial/gcc-5026.tar.gz",
+        dl_name="gcc-5026-cpp.tar.gz",
+        dest_name="gcc-5026-cpp",
+    )
+    download_tar(
+        url="https://github.com/ChrisNonyminus/powerpc-darwin-cross/releases/download/initial/gcc-5363.tar.gz",
+        dl_name="gcc-5363.tar.gz",
+        dest_name="gcc-5363",
+    )
+    download_tar(
+        url="https://github.com/ChrisNonyminus/powerpc-darwin-cross/releases/download/initial/gcc-5363.tar.gz",
+        dl_name="gcc-5363-cpp.tar.gz",
+        dest_name="gcc-5363-cpp",
+    )
+    download_tar(
+        url="https://github.com/ChrisNonyminus/powerpc-darwin-cross/releases/download/initial/gcc3-1041.tar.gz",
+        dl_name="gcc3-1041.tar.gz",
+        dest_name="gcc3-1041",
+    )
+    download_file(
+        url="https://gist.githubusercontent.com/ChrisNonyminus/ec53837b151a65e4233fa53604de4549/raw/1633f3fac3d0075051105b9b4d66891086ec27bf/convert_gas_syntax.py",
+        log_name="convert_gas_syntax.py",
```

Comment:
I don't think we should be re-downloading this file a bunch of times. We should be able to download it once and then copy it